src/js/admin-edit-profile.js

```diff
@@ -175,6 +179,142 @@ var wsuwp = wsuwp || {};
 			$( "[data-for='" + field + "']" ).eq( index ).remove();
 			$( this ).remove();
 		} );
+
+		// Surface photo collection.
+		$card.on( "click", ".photo", function() {
+			if ( $( this ).hasClass( "wsu-person-add-photo" ) ) {
+				return;
+			}
+
+			var position = $( this ).offset();
+			previous_focus = document.activeElement;
+
+			$( window ).scrollTop( 0 );
+			$( "body" ).addClass( "wsu-person-photo-collection-open" );
+
+			$photo_collection.css( {
+				"top": position.top,
+				"left": position.left
+			} );
+
+			$( ".wsu-person-add-photo" ).focus();
+		} );
+
+		// Handle keyboard interactions with the photo collection.
+		$( document ).keydown( function( e ) {
+			if ( !$( "body" ).hasClass( "wsu-person-photo-collection-open" ) ) {
+				return;
+			}
+
+			// Trap the focus within the photo collection.
+			if ( e.which === 9 ) {
+				var focusable_elements = $photo_collection.find( "button" ),
+					focused_element_index = focusable_elements.index( $( ":focus" ) );
+
+				if ( e.shiftKey && 0 === focused_element_index ) {
```

Comment:
Haha, I can't take too much credit for that idea - I borrowed it and a couple others from https://github.com/gdkraus/accessible-modal-dialog. I need to credit that properly!